neutrinoapp/src/Search.jsx

```diff
@@ -167,7 +167,7 @@ export default function Search () {
           </tbody>
         </table>
         <div className={classes.root}>
-        { !error400 ? displayWiki.length < 1 ? <p>No data found</p> : 
+        { !error400 ? displayWiki.length < 1 ? <p>No data found for {keyword}</p> : 
```

Comment:
If possible, I'd prefer an explicit if/else statement to a nested ternary if. It may be that there is some JSX reason that it has to be like this, though?

Is a further change needed to deal with requests for page number < 1?

Reply:
In  JSX, conditional rendering is done through ternary if.  Can't use if/else for this.

Reply:
I was afraid that that might be it :)

Could you insert a comment to describe the logic? I have a feeling that that might also be tricky in JSX, so no problem if not.